packages/health-check/src/checks/extended.js

```diff
@@ -5,906 +5,21 @@ const { isEqual } = require("lodash");
-          len: 76744822,
-        },
-      },
-    },
-    headers: {
-      "skynet-skylink": "AAC5glnZyNJ4Ieb4MhnYJGtID6qdMqEjl0or5EvEMt7bWQ",
-      "content-disposition": 'inline; filename="Unzip_The_Uncensored_Library_Map.zip"',
-      "content-type": "application/zip",
-    },
-  };
-
-  skylinkVerification(done, linkInfo);
-}
-
-function uncensoredLibraryPressReleaseCheck(done) {
-  const linkInfo = {
-    name: "The Uncensored Library - Press Release",
-    skylink: "AABHwuml_EhvyY8Gm7j1E2xGwodUNAJgX0A6-Cd22p9kNA",
-    bodyHash: "da39a3ee5e6b4b0d3255bfef95601890afd80709",
-    metadata: {
-      filename: "press-release-Reporters-Without-Borders-The-Uncensored-Library.zip",
-      subfiles: {
-        "press-release-Reporters-Without-Borders-The-Uncensored-Library.zip": {
-          filename: "press-release-Reporters-Without-Borders-The-Uncensored-Library.zip",
-          contenttype: "application/zip",
-          len: 383501533,
-        },
-      },
-    },
-    headers: {
-      "skynet-skylink": "AABHwuml_EhvyY8Gm7j1E2xGwodUNAJgX0A6-Cd22p9kNA",
-      "content-disposition": 'inline; filename="press-release-Reporters-Without-Borders-The-Uncensored-Library.zip"',
-      "content-type": "application/zip",
-    },
-  };
-
-  // request too large, use head just to verify the headers
-  skylinkVerification(done, linkInfo, { method: "head" });
-}
-
-function uncensoredLibraryV2Check(done) {
-  const linkInfo = {
-    name: "The Uncensored Library V2",
-    skylink: "AAAs-JOsRGWgABYIo7AwTDqSX79-BxQKjDj0wiRGoRPFnw",
-    bodyHash: "f2a802c2b7482825613a08853538203a53c96bd1",
-    metadata: {
-      filename: "The Uncensored Library V2.zip",
-      subfiles: {
-        "The Uncensored Library V2.zip": {
-          filename: "The Uncensored Library V2.zip",
-          contenttype: "application/zip",
-          len: 101262134,
-        },
-      },
-    },
-    headers: {
-      "skynet-skylink": "AAAs-JOsRGWgABYIo7AwTDqSX79-BxQKjDj0wiRGoRPFnw",
-      "content-disposition": 'inline; filename="The Uncensored Library V2.zip"',
-      "content-type": "application/zip",
-    },
-  };
-
-  skylinkVerification(done, linkInfo);
-}
-
-function bitcoinWhitepaper(done) {
-  skylinkVerification(done, {
-    name: "Bitcoin Whitepaper",
-    skylink: "3ACpC9Umme41zlWUgMQh1fw0sNwgWwyfDDhRQ9Sppz9hjQ",
-    bodyHash: "8de2fdb04edce612738eb51e14ecc426381f8ed8",
-    headers: {
-      "skynet-skylink": "3ACpC9Umme41zlWUgMQh1fw0sNwgWwyfDDhRQ9Sppz9hjQ",
-      "content-disposition": 'inline; filename="bitcoin.pdf"',
-      "content-type": "application/pdf",
-    },
+// corpusCheck runs through all the tests hosted at the resolver skylink created
+// by the testing-corpus repo https://github.com/SkynetLabs/testing-corpus
+async function corpusCheck(done) {
+  // Query the resolver skylink that the testing corpus is stored at
+  const query = `https://040e2npojpl9tiahghgls0d13bvacn5qo9jodruda8lcp3l4q3h1ikg.siasky.net/`;
+  const json = await got(query).json();
```

Comment:
we should not rely on a skylink to provide test data because if this fetch fails we will not run the full test suite - we should store that data as json inside of the repo